lib/dashboards.js

```diff
@@ -172,8 +172,8 @@ function assembleDashboard(options) {
         } else {
             return reject(NICE_ERRORS.UNEXPECTED_LIMIT_TYPE(options.limit));
         }
-        const outputFile = path.join(options.userConfig.output_directory, outputSlug);
-        const outputFileRedirect = path.join(options.userConfig.output_directory, outputSlugRedirect);
+        const outputFile = path.join(options.userConfig.output_directory, outputSlug).toLowerCase();
```

Comment:
only lowercase the slug, not the whole `outputFile` (they might be outputting into a folder named `HereIsTheDashboard`, which is entirely allowed.

(also applies to some of the other changes in this PR, of course.)

Reply:
(other than that, this fix is good.)

Reply:
I believe it only happens on lines 175, and 176.

I checked the other lines where I made changes, and they appear to be filenames only.

Let me know if anything was missed.